index.bs

```diff
@@ -1538,6 +1591,7 @@ an assertion. Its {{PublicKeyCredentialRequestOptions/challenge}} member must be
         unsigned long                        timeout;
         USVString                            rpId;
         sequence<PublicKeyCredentialDescriptor> allowCredentials = [];
+        UserVerificationRequirement          userVerification = "preferred";
```

Comment:
ditto

Reply:
IMO, Default should be to allow both kind of authenticators (one with UV and one without).

Reply:
I do not understand your comment @akshayku -- please explain?

Reply:
Out of `required` (RP requires UV), `preferred` (RP supports both kind of authenticators, one who sends UV bit set and one who don't ) and `discouraged` (RP don't want friction of user verification via ClientPin entry or otherwise but OK if he gets user verification anyway), I like `preferred`.

Reply:
ought to also get consensus with @jcjones, @balfanz & @leshi, @rlin1